### DISCUSSION TITLE
IDDM v1.1.6

### DISCUSSION THREAD

**Discussion by chairc** [chairc/Integrated-Design-Diffusion-Model#87] [Announcements]
## What's Changed
* Update: Refactor web.py, and add the generate page. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/75
* Modify custom images length and width input; Other code enhancement changes by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/76
* Add custom parameter settings; Modify the method import path and eliminate the magic value by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/77
* Update: Modify the format and delete useless import by @bestl1fe in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/78
* Update about loss function. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/79
* Update: Update the README. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/80
* Modify check_and_create_dir function; Modify comment. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/82
* Fix: Fix the problem of lack parameters when input --image_size. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/83
* Add: Add v1.1.6 tag. by @chairc in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/86

## New Contributors
* @bestl1fe made their first contribution in https://github.com/chairc/Integrated-Design-Diffusion-Model/pull/78

**Full Changelog**: https://github.com/chairc/Integrated-Design-Diffusion-Model/compare/v1.1.5...v1.1.6

## Weights
**Note: The weight include model, ema_model and optimizer.**

* `celebahq-120-weight.pt`: Trained on a dataset of 30,000 people face, and image size is 120 ([celebahq-120-weight.pt](https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/download/v1.1.5/celebahq-120-weight.pt))
* `animate-ganyu-120-weight.pt`: Trained on a dataset of 500 animate ganyu face, and image size is 120 ([animate-ganyu-120-weight.pt](https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/download/v1.1.5/animate-ganyu-120-weight.pt))
* `neu-120-weight.pt`: Trained on a dataset of 1,800 defect, and image size is 120 ([neu-120-weight.pt](https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/download/v1.1.5/neu-120-weight.pt))
* `cifar-64-weight.pt`: Trained on a dataset of 60,000 images, and image size is 64 ([cifar10-64-weight.pt](https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/download/v1.1.5/cifar10-64-weight.pt))
* `animate-face-64-weight.pt`: Trained on a dataset of 63,565 animate face, and image size is 64 ([animate-face-64-weight.pt](https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/download/v1.1.5/animate-face-64-weight.pt))

<hr /><em>This discussion was created from the release <a href='https://github.com/chairc/Integrated-Design-Diffusion-Model/releases/tag/v1.1.6'>IDDM v1.1.6</a>.</em>
